thirdparty/cmdconfig/commands/cmdsource/cmdsource.go

```diff
@@ -29,6 +33,8 @@ func GetSourceRunner(name string) *SourceRunner {
 	}
 	c.Flags().StringVar(&r.FunctionConfig, "fn-config", "",
 		"path to function config file.")
+	c.Flags().BoolVar(&r.IncludeMetaResources,
```

Comment:
Pl. update the reference docs under `site/..../fn` to add this flag.

Reply:
done